### DISCUSSION TITLE
Theming Glances Web UI?

### DISCUSSION THREAD

**Discussion by Madelena** [nicolargo/glances#2046] [Q&A] [answered]
I'm wondering if there is a way to attach CSS files to the Glances Web UI, in order to theme it? It would be nice to be able to customize the fonts and colors somehow.

**Comment by nicolargo** [comment, marked as answer]
Hi @Madelena !

There is no theme management for the WebUI, but it is a nice proposal !

As a workaround, you can change the current theme from the following folder: https://github.com/nicolargo/glances/tree/develop/glances/outputs/static/css
And rebuild the Web UI (with the **make webui** command line).

If you have some time to change the look&feel of the WebUI, any contribution will be welcome :)

Nicolas
